### ISSUE TITLE
[Bug] ha-bambulab v2.0.22 generates warnings in HA 2024.8.3

### ISSUE THREAD

**Issue by maruel** [greghesp/ha-bambulab#585]
### Describe the bug

- Incorrect casing in FanEntityFeature, turn_on instead of TURN_ON
- Fan Entities are not named, so logs is as "None"
- Need to start using async_forward_entry_setups() instead of async_forward_entry_setup() at custom_components/bambu_lab/coordinator.py, line 225 before June 2025 (2025.6).

### To Reproduce

Configure ha-bambulab  v2.0.22 in HA 2024.8.3 with a X1C. Look at warning level logs in HA.

### Expected Behaviour

No warnings in HA's logs.

### What device are you using?

X1C

### Diagnostic Output

```shell
See logs.
```

### Log Extracts

```shell
2024-09-02 09:53:15.852 WARNING (MainThread) [homeassistant.components.fan] Entity None (<class 'custom_components.bambu_lab.fan.BambuLabFan'>) does not set FanEntityFeature.TURN_OFF but implements the turn_off method. Please create a bug report at https://github.com/greghesp/ha-bambulab/issues
2024-09-02 09:53:15.852 WARNING (MainThread) [homeassistant.components.fan] Entity None (<class 'custom_components.bambu_lab.fan.BambuLabFan'>) does not set FanEntityFeature.TURN_ON but implements the turn_on method. Please create a bug report at https://github.com/greghesp/ha-bambulab/issues
2024-09-02 09:53:15.852 WARNING (MainThread) [homeassistant.components.fan] Entity None (<class 'custom_components.bambu_lab.fan.BambuLabFan'>) does not set FanEntityFeature.TURN_OFF but implements the turn_off method. Please create a bug report at https://github.com/greghesp/ha-bambulab/issues
2024-09-02 09:53:15.852 WARNING (MainThread) [homeassistant.components.fan] Entity None (<class 'custom_components.bambu_lab.fan.BambuLabFan'>) does not set FanEntityFeature.TURN_ON but implements the turn_on method. Please create a bug report at https://github.com/greghesp/ha-bambulab/issues
2024-09-02 09:53:15.852 WARNING (MainThread) [homeassistant.components.fan] Entity None (<class 'custom_components.bambu_lab.fan.BambuLabFan'>) does not set FanEntityFeature.TURN_OFF but implements the turn_off method. Please create a bug report at https://github.com/greghesp/ha-bambulab/issues
2024-09-02 09:53:15.852 WARNING (MainThread) [homeassistant.components.fan] Entity None (<class 'custom_components.bambu_lab.fan.BambuLabFan'>) does not set FanEntityFeature.TURN_ON but implements the turn_on method. Please create a bug report at https://github.com/greghesp/ha-bambulab/issues
2024-09-02 09:53:16.245 WARNING (MainThread) [homeassistant.helpers.frame] Detected that custom integration 'bambu_lab' calls async_forward_entry_setup for integration, bambu_lab with title: 00M00A282504424 and entry_id: c0fd24a458b472e3ccadf3327964b83f, which is deprecated and will stop working in Home Assistant 2025.6, await async_forward_entry_setups instead at custom_components/bambu_lab/coordinator.py, line 225: await self.hass.config_entries.async_forward_entry_setup(self.config_entry, Platform.SENSOR), please create a bug report at https://github.com/greghesp/ha-bambulab/issues
```

### Other Information

Thanks a lot for this integration, it's amazing!

**Comment by vordenken** [comment]
Came here to report this as well :) +1

**Comment by ErikvO** [comment]
See [https://developers.home-assistant.io/blog/2024/07/19/fan-fanentityfeatures-turn-on_off/](this developer post) for more information.

**Comment by ErikvO** [comment]
This issue is a duplicate of #573 and fixed in #583
